launcher/package.json

```diff
@@ -33,6 +33,7 @@
     "electron-store": "^8.1.0",
     "electron-updater": "^5.2.1",
     "file-saver": "^2.0.5",
+    "js-yaml": "^4.1.0",
```

Comment:
pls remove

Reply:
@MaxTheGeeek I think it is used to load .zip file right?